biogtr/models/global_tracking_transformer.py

```diff
@@ -0,0 +1,127 @@
+"""Module containing GTR model used for training."""
+from biogtr.models.transformer import Transformer
+from biogtr.models.visual_encoder import VisualEncoder
+from torch import nn
+
+# todo: do we want to handle params with configs already here?
+
+
+class GlobalTrackingTransformer(nn.Module):
```

Comment:
Do we also handle high level instantiation via hydra? maybe this is already added in subsequent PRs

Reply:
Yeah the GTR is instantiated within GTRRunner which is handled by the hydra config